andela/hc-shleemypants
# 155965545-enable check prioritization

**What does this PR do?**

- Enables assigning different priorities to checks.

**Description of Task to be completed?**

- A user should be able to assign different priorities to checks i.e emergency, high, normal, low or lowest.


**How should this be manually tested?**

- Pull the latest changes to this branch, create a check and it should have a priority button which defaults to normal. assign the check a different priority and it should reflect.

**How should this be manually tested?**

- 155965545

**Screenshots**
<img width="930" alt="priority" src="https://user-images.githubusercontent.com/17777137/38795044-b154dd3c-415f-11e8-9381-b40132a343b3.png">
<img width="1017" alt="sample priorities" src="https://user-images.githubusercontent.com/17777137/38795178-0eed3a20-4160-11e8-9c7c-795c3d447223.png">
